yeat/workflow/__init__.py

```diff
@@ -17,14 +17,28 @@ def run_workflow(args):
     snakefile = files("yeat") / "workflow" / "Snakefile"
     config = vars(args)
     config["data"] = resolve_paths(config["config"])
-    success = snakemake(
-        snakefile,
-        config=config,
-        cores=args.threads,
-        dryrun=args.dry_run,
-        printshellcmds=True,
-        workdir=args.outdir,
-    )
+    if args.grid:
+        success = snakemake(
+            snakefile,
+            config=config,
+            cores=args.threads,
+            dryrun=args.dry_run,
+            printshellcmds=True,
+            workdir=args.outdir,
+            local_cores=args.threads,
+            nodes=args.grid_limit,
+            drmaa=setup_grid_args(args),
+            drmaa_log_dir=str((Path(args.outdir) / "gridlogs").resolve()),
```

Comment:
In order to save the DRMAA snakemake log files, this must be a full path and not relative.

Reply:
Good to know!